src/client/src/ui/analytics/views/analyticsBarChart.jsx

```diff
@@ -9,19 +10,29 @@ export default class PNLChart extends React.Component{
     isPnL: React.PropTypes.bool
   };
 
-  _createBars(){
+  render(){
+    let bars = this._createBars();
+    return (
+      <div>{bars}</div>
+    );
+  }
 
+  _createBars(){
     const containerWidth = 330;
+
     let propName = this.props.isPnL ? CurrencyPairPosition.basePnlName : CurrencyPairPosition.baseTradedAmountName;
-    let maxMinValues = this.props.series.reduce( (resultObj, element) => {
+    let chartData = this.props.isPnL ? this.props.series : this._getPositionsDataFromSeries(propName);
+
+    let maxMinValues = chartData.reduce( (resultObj, element) => {
       resultObj.max = element[propName] > resultObj.max ? element[propName] : resultObj.max;
```

Comment:
why not `resultObj.min = Math.max(element[propName], resultObj.max)` and `... Math.min(element[propName], resultObj.min);`

you could even do:

```js
let vals = _.map(this.props.series, propName),
        max = _.max(vals),
        min = _.min(vals);
```    - this will be x2 faster than reduce with an arrow function.